Gemfile

```diff
@@ -11,6 +11,8 @@ gem 'jquery-rails'
 
 gem 'dotenv-rails', :groups => [:development]
 
+gem 'use_case_pattern'
```

Comment:
We can remove the gem for now. When removed run `bundle install` again and the commit the changes